pubspec.yaml

```diff
@@ -16,6 +16,7 @@ dev_dependencies:
   dart_dev: "^1.0.0"
   dart_style: "^0.2.1"
   dartdoc: "^0.8.0"
+  logging: "^0.11.0"
```

Comment:
Haha don't worry, logging is pretty universal. And I'm about to add another dependency anyway (meta)